src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -36,7 +35,8 @@ private static SafeTerminateHandler<TraceEventSession> MakeSafeTerminateTraceEve
         /// <returns></returns>
         public static void Record(string assemblyFileName, string runId, string outputDirectory, Action action, Action<string> collectOutputFilesCallback)
         {
-            if (!IsRunningAsAdministrator)
+            bool? isElevated = TraceEventSession.IsElevated();
+            if (!isElevated.HasValue || isElevated.Value == false)
```

Comment:
I'm curious, what does `null` mean in this context? What happens if we're running on a machine where UAC is completely turned off?